src/components/connect/social-media-links.svelte

```diff
@@ -3,26 +3,25 @@
   import DiscordIcon from "@/components/icons/discord.svelte";
   import LinkedInIcon from "@/components/icons/linkedin.svelte";
   import YouTubeIcon from "@/components/icons/youtube.svelte";
-  import { links } from "../../lib/links";
```

Comment:
Good simplification here.

Reply:
Wanted to make use of our custom URLs as a subtle touch to make the site more polished, but i'll reimplement them if you really want it

Reply:
What you did works great to me 😁